java/ql/src/Security/CWE/CWE-200/TempDirLocalInformationDisclosure.ql

```diff
@@ -0,0 +1,258 @@
+/**
+ * @name Temporary Directory Local information disclosure
```

Comment:
I'm wondering if this might be clearer. The query name should be in sentence case.

```suggestion
 * @name Local information disclosure in a temporary directory
```